circle.yml

```diff
@@ -39,7 +44,7 @@ defaults:
     run:
       name: "basic CLI validation"
       command: |
-        ./bin/evm2wasm.js asdfdsdf --trace
+        ./bin/evm2wasm.js 600160020200 --trace
```

Comment:
Do these have `set -e` enabled? e.g. stopping on any error

Reply:
`./bin/evm2wasm.js asdfdsdf --trace` this doesn't produce an error.

Reply:
https://github.com/ewasm/evm2wasm/issues/223